### ISSUE TITLE
app:fill-resourcespace command crashes if it detects a subdirectory

### ISSUE THREAD

**Issue by netsensei** [VlaamseKunstcollectie/Imagehub#4]
**Problem:**

When a directory is present in the "images" folder, the app:fill-resourcespace command will crash with an error:

```
ImagickException: The path is a directory: /iiif-data/lost+found in /srv/imagehub/imagehub/src/AppBundle/ResourceSpace/Command/FillResourceSpaceCommand.php:82
```

Logic would dictate that the `images` directory doesn't contain any subdirectories. However, on the testbed VPS, the our images directory is actually `/iiif-data` which is a mount point for a different disk volume. This disk volume contains a standard UNIX `lost+found` directory we can't delete. 

So, we end up with this error message:

```
ImagickException: The path is a directory: /iiif-data/lost+found in /srv/imagehub/imagehub/src/AppBundle/ResourceSpace/Command/FillResourceSpaceCommand.php:82
```

**Solution**

use `is_file()` or `is_dir()` functions to detect whether or not an item from the `scandir()` function is a directory or a file. Omit directories, only process files.

**Comment by Kitania** [comment]
Fixed in 45dcb6b

**Comment by Hobbesball** [comment]
reviewed and approved.
